Gemfile

```diff
@@ -60,3 +60,5 @@ end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+
+gem 'active_model_serializers'
```

Comment:
minor:
you can setup `rubocop` - the ruby linter.
help you with small things and best practices and norms of ruby. (like new line at end of file, "magic comment"/frozen literals at the start of the file)